packages/person/src/components/select/element.ts

```diff
@@ -172,64 +181,78 @@ export class PersonSelectElement
    */
   protected renderList(): HTMLTemplateResult {
     if (!this.controllers.element.isOpen) {
+      if (this.selectedPerson && this.tasks.info.status === 2) {
```

Comment:
Can you use the `TaskStatus.COMPLETE` instead? I don't like magic numbers.

Reply:
Done